docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_4.rst

```diff
@@ -12,6 +12,10 @@ Release notes: Open GEE 5.3.4
 
       .. rubric:: New Features
 
+      **Optimized handling of status updates for Canceled and Failed tasks**.
+      System manager's algorithm for handling updates for Canceled and Failed

```

Comment:
Very minor: I think "updates **to**" might work better than "updates **for**"

Reply:
I don't disagree, but they feel fairly similar, so I'm inclined to leave it as is.